src/chat_sdk/conversations/private_v1.nim

```diff
@@ -127,7 +145,7 @@ proc wireCallbacks(convo: PrivateV1, deliveryAckCb: proc(
 
 
 proc initPrivateV1*(owner: Identity, participant: PublicKey, seedKey: array[32, byte],
-        discriminator: string = "default", isSender: bool, deliveryAckCb: proc(
+        discriminator: string = "default", inviter: bool, deliveryAckCb: proc(
```

Comment:
I think moving away from the "isSender" convention makes sense here.

[Sand] Perhaps "Initiator" would be a better fit as it keeps the language consistent.

Reply:
When reading the proto definition, I somehow need to think twice to reason the term `participant` (`initiator` seems fine though). The term inviter/invitee looks a bit clear and straightforward to me.
```
message InvitePrivateV1 {
    bytes initiator = 1;
    bytes initiator_ephemeral = 2;
    bytes participant = 3;
    int32 participant_ephemeral_id= 4;
    string discriminator = 5;
}
```

Reply:
Clear unambiguous terms are important. Feel free to suggest new ones here: https://github.com/waku-org/specs/pull/80